pipelines/DRP.yaml

```diff
@@ -160,6 +161,45 @@ subsets:
       The multiVisit subset defined in pipe_tasks' DRP.yaml is not safe to
       use on HSC for various reasons; use 'step1', 'step2', and 'step3' subsets
       instead.  It may be re-enabled in the future.
+  faro:
```

Comment:
Would it make sense to categorize faro tasks as faro_step1, faro_step2 etc. based on when they can be run and faro_step1 can be included in step1? If that is not possible to merge before the weekly deadline, please go ahead and merge but you could consider reorganization later. The reason I ask is that it'd make it easier to add new metrics/tasks later on as we add them.